code/modules/reagents/Chemistry-Recipes/Others.dm

```diff
@@ -124,6 +126,40 @@
 			D.RemoveSymptom(healer)
 			D.AddSymptom(new /datum/symptom/regen())
 			D.Refresh(1) //It needs to actually change the disease name for reasons
+		//scarab symptom
+		var/itching = locate(/datum/symptom/itching) in D.symptoms
+		var/hallucinogen = locate(/datum/symptom/hallucigen) in D.symptoms
+		var/vomit = locate(/datum/symptomvomit/blood) in D.symptoms
+		if(itching && hallucinogen && vomit)
+			D.RemoveSymptom(itching)
+			D.RemoveSymptom(hallucinogen)
+			D.RemoveSymptom(vomit)
+			D.AddSymptom(new /datum/symptom/scarab())
+			D.Refresh(1) //It needs to actually change the disease name for reasons
+		//explosive death
+		var/choking = locate(/datum/symptom/choking) in D.symptoms
+		var/asthmothia = locate(/datum/symptom/asthmothia) in D.symptoms
+		if(choking && asthmothia)
+			D.RemoveSymptom(choking)
+			D.RemoveSymptom(asthmothia)
+			D.AddSymptom(new /datum/symptom/explosive())
+			D.Refresh(1)
+		//purge virus
+		var/liggeritis = locate(/datum/symptom/liggeritis) in D.symptoms
+		var/metabolism = locate(/datum/symptom/heal/metabolism) in D.symptoms
+		if(liggeritis && metabolism)
+			D.RemoveSymptom(liggeritis)
+			D.RemoveSymptom(metabolism)
+			D.AddSymptom(new /datum/symptom/purge())
+			D.Refresh(1)
+		//sensory restoration
+		var/visionaid = locate(/datum/symptom/visionaid) in D.symptoms
+		var/youth = locate(/datum/symptom/heal/youth) in D.symptoms
```

Comment:
Please double check this typepath.

code/modules/reagents/Chemistry-Recipes/Others.dm:157:error: /datum/symptom/heal/youth: undefined type path